winbuild/build_prepare.py

```diff
@@ -116,7 +119,7 @@ def cmd_msbuild(
     "HARFBUZZ": "11.1.0",
     "JPEGTURBO": "3.1.0",
     "LCMS2": "2.17",
-    "LIBAVIF": "1.2.1",
+    "LIBAVIF": "1.3.0",
```

Comment:
Please also change the libavif version to 1.3.0 in Pillow/depends/install_libavif.sh.